pkg/models/quotas.go

```diff
@@ -56,10 +58,11 @@ type GetOrgQuotasQuery struct {
 }
 
 type GetUserQuotaByTargetQuery struct {
-	Target  string
-	UserId  int64
-	Default int64
-	Result  *UserQuotaDTO
+	Target           string
+	TargetConditions string
```

Comment:
Think you can skip adding `TargetConditions` for `GetUserQuotaByTargetQuery`. Can be added in the future, if needed

Reply:
Yeah, wasn't sure on whether to do this or not. Sounds good to me.